tests/DqtApi.Tests/UnitTests/TeachersController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using DqtApi.DAL;
+using DqtApi.Models;
+using DqtApi.Responses;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Xunit;
+
+namespace DqtApi.Tests.UnitTests
+{
+    public class TeachersController
```

Comment:
Could we merge the tests here with `GetTeacherTests`? There seems to be some cross-over.

Reply:
Not sure what cross-over there is?  DataverseIntegration\GetTeacherTests tests the DataverseAdaptor - I'll rename it to DataverseAdaptorTests.  DqtApi.FunctionalTests\Endpoints\GetTeacherTests tests end-to-end functionality.